ai/question.py

```diff
@@ -57,16 +57,19 @@ def ask(repo_id: int, chat_id: str, question: str):
 
     memory = _get_chat_memory(chat_id)
 
-    qa = ConversationalRetrievalChain.from_llm(
-        llm=ChatOpenAI(temperature=0),
+    agent = ConversationalRetrievalChain.from_llm(
+        llm=ChatOpenAI(temperature=0, model='gpt-3.5-turbo-16k'),
         memory=memory,
         retriever=retriever,
         return_source_documents=True,
     )
 
-    result = qa(question)
+    result = agent({'question': question})
```

Comment:
@aaa006bd Why do you want to do this change at all? It would skip the history IMO. Did you tested it?

Reply:
@anbraten otherwise it starts throwing an error for some reason. I am having another issue regarding the migration. Before I tested it. But you can test and if they are the same please revert the changes.